src/main/scala/mesosphere/marathon/core/task/bus/impl/TaskStatusEmitterImpl.scala

```diff
@@ -11,7 +11,7 @@ private[bus] class TaskStatusEmitterImpl(internalTaskStatusEventStream: Internal
 
   override def publish(taskChanged: TaskChanged): Unit = {
     taskChanged.stateOp match {
-      case TaskStateOp.MesosUpdate(task, status, timestamp) =>
+      case TaskStateOp.MesosUpdate(task, marathonTaskStatus, mesosStatus, timestamp) =>
```

Comment:
since we don't use any of the extracted vals here, this could be `case update: TaskStateOp.MesosUpdate =>`
(this avoids extracting the values and assigning them)